libraries/SrcWrapper/inc/uart.h

```diff
@@ -138,6 +138,9 @@ struct serial_s {
 #elif defined(STM32U0xx)
 #define USART3_IRQn USART3_LPUART1_IRQn
 #define USART3_IRQHandler USART3_LPUART1_IRQHandler
+#elif defined(STM32C0xx)
+#define USART3_IRQn USART3_4_IRQn
+#define USART3_IRQHandler USART3_4_IRQHandler
```

Comment:
This could by simplified by adding STM32C0xx with STM32G0xx
`#elif defined(STM32G0xx) || defined(STM32C0xx)`